extensions/eda/plugins/event_source/generic.py

```diff
@@ -99,6 +104,10 @@ def __init__(self: Generic, queue: asyncio.Queue, args: dict[str, Any]) -> None:
         """Insert event data into the queue."""
         self.queue = queue
         field_names = [f.name for f in fields(Args)]
+
+        if "payload_file" in args:
+            args["payload"] = ""
```

Comment:
This should not be an empty array? Which also makes me notice that the type of "payload" is wrong in the dataclass.

Reply:
It can be a simple string which gets converted to an array.

Reply:
ok, I see.